### ISSUE TITLE
Blank screen after updating

### ISSUE THREAD

**Issue by KK-Designs** [spicetify/spicetify-cli#2558]
### 🔍 Have you checked Spicetify.app page for your issue?

- [X] I have checked the FAQ

### 🔍 Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### ℹ Environment / Computer Info

```Markdown
- Spotify version: 1.2.20.1210.g2a8a8a57
- Spicetify version: 2.23.1
```

### 📝 Description

After I updated Spotify and Spicetify and applied the backup, my Spotify now shows a blank screen. Rolling back the Spotify version does not fix the issue. Do you have any idea why it's broken? Running `spicetify restore` makes Spotify work but none of the themes or plugins are active.

## Blank screen
![image](https://github.com/spicetify/spicetify-cli/assets/71038229/12864632-f9a9-447b-813f-fc10cde525e4)

## Dev Console
![image](https://github.com/spicetify/spicetify-cli/assets/71038229/bdce80d8-af87-4ebb-93be-f0d6fc352064)

**Comment by dhupee** [comment]
I believe this issue is being fixed with #2556

check the release note as well

**Comment by rxri** [comment]
Duplicate of #2555
